source/TestFrameworkShared/Helper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+
+namespace nanoFramework.TestFramework
+{
+    public static class Helper
+    {
+        private static string GetJoinedParams(object[] data)
+        {
+            var returnString = string.Empty;
+            foreach (var item in data)
+            {
+                returnString += $"{item} | ";
+            }
+
+            return returnString.Substring(0, returnString.Length - 4);
+        }
+
+        public static string GetDisplayName(MethodInfo method, object attribute)
```

Comment:
missing intellisence comments

Reply:
I've changed the method name to GetTestDisplayName and the comment is there :)

        /// <summary>
        /// Generates test display name based on passed <paramref name="method"/> and <paramref name="attribute"/>. 
        /// </summary>
        /// <returns>Returns method name with parameters if passed attribute is of DataRow type</returns>
        public static string GetTestDisplayName(MethodInfo method, object attribute)